lxd/certificates.go

```diff
@@ -636,9 +636,6 @@ func certificatePut(d *Daemon, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 
-	// Expand the fingerprint.
-	fingerprint = oldEntry.Fingerprint
```

Comment:
What happens if I use a partial fingerprint such as the one we show in `lxc cluster list`?

From what I remember, only `GetCertificate` allows using a short fingerprint, all other functions expect the full fingerprint, so this change most likely breaks anyone interacting with LXD using the fingerprint we should in `lxc cluster list`.

Reply:
> From what I remember, only `GetCertificate` allows using a short fingerprint, all other functions expect the full fingerprint

That's incorrect. Both `PATCH` and `PUT` support partial fingerprints.

> so this change most likely breaks anyone interacting with LXD using the fingerprint we should in `lxc cluster list`.

This doesn't break anything. It only removes the unnecessary `fingerprint` variable as it's equal to `oldEntry.Fingerprint`.